androidApp/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ android {
         targetSdk = 34
         versionCode =
             Integer.parseInt((findProperty("android.injected.version.code") ?: "1") as String)
-        versionName = (findProperty("android.injected.version.name") ?: "0.1.0") as String
+        versionName = "0.2.0"
```

Comment:
This has been sitting at 0.1.0 since we started; this seems like as reasonable a time as any to bump it up a bit.

I'm not actually sure why this had been defined in Fastlane before.